src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -220,11 +222,15 @@ function DesktopTriggers() {
           [styles['has-multiple-triggers']]: hasMultipleTriggers,
           [styles['has-open-drawer']]: hasOpenDrawer,
         })}
+        role="toolbar"
+        aria-orientation="vertical"
       >
         {visibleItems.map(item => {
           return (
             <TriggerButton
               ariaLabel={item.ariaLabels?.triggerButton}
+              ariaExpanded={item.id === activeDrawerId}
+              ariaControls={item.id}
```

Comment:
So, when we select a drawer, we set it's `item.id` as an `activeDrawerId`, right? In that case all other trigger buttons will point to a non-existing element in the DOM, which is not good.

I think we need something like `ariaControls={item.id === activeDrawerId ? item.id : undefined}`, but we can double-check.

The same applies to other ariaControls :)

Reply:
I think this makes a lot of sense! Doesn't make sense to have a control for something that doesn't exist (see my other comment around mobile behavior)